docs/tutorials/Remark.adoc

```diff
@@ -0,0 +1,396 @@
+public RemarkCommand parse(String args) throws ParseException {
+    requireNonNull(args);
+    ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args,
+        PREFIX_REMARK);
+
+    Index index;
+    try {
+        index = ParserUtil.parseIndex(argMultimap.getPreamble());
+    } catch (IllegalValueException ive) {
+        throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+            RemarkCommand.MESSAGE_USAGE), ive);
+    }
+
+    String remark = argMultimap.getValue(PREFIX_REMARK).orElse("");
+
+    return new RemarkCommand(index, remark);
+}
+----
+
+NOTE: Don't forget to update `AddressBookParser` to use our new `RemarkCommandParser`!
+
+If you are stuck, check out the sample link:https://github.com/j-lum/addressbook-level35/commit/c0f8419be4fb59bffafcd73026769459e5cf5c5e#diff-fc19ecee89c3732a62fbc8c840250508[here].
+
+== Add `Remark` to the model
+
+Now that we have all the information that we need, let's lay the groundwork for some _persistent_ changes.
+We achieve that by working with the `Person` model.
+Each field in a Person is implemented as a separate class (e.g. a `Name` object represents the person's name).
+That means we should add a `Remark` class so that we can use a `Remark` object to represent a remark given to a person.
+
+=== Add a new `Remark` class
+
+Create a new `Remark` in `seedu.address.model.person`. Since a `Remark` is a field that is similar to `Address`, we can reuse a significant bit of code.
+
+A quick copy-paste and search-replace later, you should have something like link:https://github.com/j-lum/addressbook-level35/commit/c8e6048f0d81f2b4b7797e838226da823892129b#diff-af2f075d24dfcd333876f0fbce321f25[this].
+
+=== Make use of `Remark`
+
+Let's change `RemarkCommand` and `RemarkCommandParser` to use the new `Remark` class instead of plain `String`.
+These should be relatively simple changes.
+
+== Add a placeholder element for remark to the UI
+
+Without getting too deep into `fxml`, let's go on a 5 minute adventure to get some placeholder text to show up for each person.
+
+Simply add
+[source, java]
+PersonCard.java
+```
+@FXML
+private Label remark;
+```
+
+to link:https://github.com/j-lum/addressbook-level35/commit/06f9897d7088d22689de478c94612f8bcbff2964#diff-0c6b6abcfac8c205e075294f25e851fe[`seedu.address.ui.PersonCard`].
+`@FXML` is an annotation that marks a private or protected field and makes it accessible to FXML.
+It might sound like Greek to you right now, don't worry -- we will get back to it later.
+
+Then insert
+
+```
+<Label fx:id="remark" styleClass="cell_small_label" text="\$remark" />
+```
+into link:https://github.com/j-lum/addressbook-level35/commit/06f9897d7088d22689de478c94612f8bcbff2964#diff-12580431f55d7880578aa4c16f249e71[`main/resources/view/PersonListCard.fxml`].
+
+That's it! Fire up the application again and you should see something like this:
+
+.$remark shows up in each entry
+image::$Remark.png[]
+
+== Modify `Person` to support a `Remark` field
```

Comment:
Doing this first thing in the morning after eating breakfast, my **personal** opinion is that **Modify Person to support a Remark field** gotta be the hardest part of the tutorial.

You've been giving commit links for people to reference to, to not provide one all of a sudden is quite jarring. I think the relevant commit is this (please double-check)
https://github.com/j-lum/addressbook-level3/commit/b03b788d5a462403ea7f4dce9a832b62f729f7c4

Reply:
Yeah some parts of the tutorial are really short in text and intense in execution because it's pretty much impossible to guide them through with code snippets. Linked the commit to save them some pain.